openai/src/commonMain/kotlin/com/xebia/functional/xef/conversation/llm/openai/Conversions.kt

```diff
@@ -0,0 +1,59 @@
+package com.xebia.functional.xef.conversation.llm.openai
+
+import com.aallam.openai.api.chat.ChatChunk as OAIChatChunk
+import com.aallam.openai.api.chat.ChatCompletionChunk as OAIChatCompletionChunk
+import com.aallam.openai.api.chat.ChatDelta as OAIChatDelta
+import com.aallam.openai.api.chat.ChatRole
+import com.aallam.openai.api.core.Usage as OAIUsage
+import com.xebia.functional.xef.llm.models.chat.ChatChunk
+import com.xebia.functional.xef.llm.models.chat.ChatCompletionChunk
+import com.xebia.functional.xef.llm.models.chat.ChatDelta
+import com.xebia.functional.xef.llm.models.chat.Role
+import com.xebia.functional.xef.llm.models.functions.FunctionCall
+import com.xebia.functional.xef.llm.models.usage.Usage
+
+internal fun ChatRole.toInternal() =
+  when (this) {
+    ChatRole.User -> Role.USER
+    ChatRole.Assistant -> Role.ASSISTANT
+    ChatRole.System -> Role.SYSTEM
+    ChatRole.Function -> Role.SYSTEM
```

Comment:
Do we need a Function Role in Xef or do we want a System Role here?

If we are going to address this in another PR, maybe we can write a comment here and add a new issue in the repo

Reply:
I have added a new issue for that #443